packages/vega-typings/types/spec/encode.d.ts

```diff
@@ -81,27 +81,75 @@ export interface ColorHCL {
   l: NumericValueRef;
 }
 export interface GradientStop {
+  /**
+   * The offset fraction for the color stop, indicating its position within the gradient.
+   */
   offset: number;
-  color: string;
+  /**
+   * The color value at this point in the gradient.
+   */
+  color: Color;
 }
 export interface GradientLinear {
+  /**
```

Comment:
Better have GradientLinear/ a generic gradient type so that docs for type can describe both types. 

```
gradient: 'linear' | 'radial'  // could be a generic T that extends 'linear' | 'radial' too 
```

Reply:
Done